### PR TITLE
[zh-cn]A-few-small-changes-in-volumes

### PR DESCRIPTION
**Pull request by yanrongshi**
Fllow up to :https://kubernetes.io/docs/concepts/storage/volumes/#downwardapi
image

### CODE REVIEW

File: content/zh-cn/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -540,11 +540,11 @@ It mounts a directory and writes the requested data in plain text files.
...
 
 <!--
-A Container using Downward API as a [subPath](#using-subpath) volume mount will not
-receive Downward API updates.
+A container using the downward API as a [`subPath`](#using-subpath) volume mount does not
+receive updates when field values change.
 -->
 {{< note >}}
-容器以 [subPath](#using-subpath) 卷挂载方式使用 downwardAPI 时，将不能接收到它的更新。
+容器以 [subPath](#using-subpath) 卷挂载方式使用 downwardAPI 时，在字段值更改时将不能接收到它的更新。
```
Comment (ydFu) [inline content/zh-cn/docs/concepts/storage/volumes.md:547]:
```suggestion
容器以 [`subPath`](#using-subpath) 卷挂载方式使用 downward API 时，在字段值更改时将不能接收到它的更新。
```

Comment (yanrongshi) [reply content/zh-cn/docs/concepts/storage/volumes.md]: Thanks,LGTM

Comment (ydFu) [reply content/zh-cn/docs/concepts/storage/volumes.md]: > ```suggestion
> 
> 容器以 [`subPath`](#using-subpath) 卷挂载方式使用 downward API 时，在字段值更改时将不能接收到它的更新。
> 
> ```

Miss Markdown of code block for subPath.
You can refer
https://www.markdownguide.org/basic-syntax/#code

Thanks!

**Pull request merged**